### ISSUE TITLE
Missing docs for > [!Video] 

### ISSUE THREAD

**Issue by daxianji007** [dotnet/docfx#1865]
DFM support grammar of
```
 > [!Video]
```

Please add docs for it

**Comment by roel4ez** [comment]
I have tried out this feature, but cannot get it work completely unfortunately. 
Using the following snippet:

`> [!Video https://www.youtube.com/watch?v=RtnepXu64YQ]`

Generates the following HTML:

```html
<div class="embeddedvideo">
   <iframe src="https://www.youtube.com/watch?v=RtnepXu64YQ" frameborder="0" allowfullscreen="true">
   </iframe>
</div>
```
However, this request is blocked by Chrome:

> Refused to display 'https://www.youtube.com/watch?v=RtnepXu64YQ' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

Using a local file from the `videos` folder (after adding it in docfx.json as a resource) works _only_ if the video is referenced by full URL:
 `> [!Video http://localhost/videos/test.mp4]

It would be great if the videos can also be referenced by relative path:
`> [!Video ../videos/test.mp4]`

Should I file a new issue for this?
